test/integration/framework/test_server.go

```diff
@@ -59,7 +60,7 @@ type TearDownFunc func()
 
 // StartTestServer runs a kube-apiserver, optionally calling out to the setup.ModifyServerRunOptions and setup.ModifyServerConfig functions
 func StartTestServer(t *testing.T, setup TestServerSetup) (client.Interface, *rest.Config, TearDownFunc) {
-	certDir, err := os.MkdirTemp("", "test-integration-"+t.Name())
+	certDir, err := os.MkdirTemp("", "test-integration-"+strings.ReplaceAll(t.Name(), "/", "_"))
```

Comment:
Ugh 😩

Reply:
It took me a bit to debug it - for tests that have subcases using t.Run(...), their names contain "/", and for those creating directory was failing - seems like the simplest fix.